asab/application.py

```diff
@@ -589,10 +590,15 @@ def time(self):
 	# Housekeeping
 
 	def _set_housekeeping_time_from_config(self):
```

Comment:
this method is not technically a setter. so maybe `_compute_housekeeping_time_from_config`?